samcli/commands/init/__init__.py

```diff
@@ -231,6 +231,14 @@ def wrapped(*args, **kwargs):
     """ """,
     required=False,
 )
+@click.option(
+    "-t",
+    "--tracing",
+    default="disable",
+    type=click.Choice(["enable", "disable"]),
+    help="Enable AWS X-Ray tracing for your lambda functions",
```

Comment:
We can check with UX or documentation team whether we need more information about this flag.